src/containers/Dashboard/Quizzes/QuizCard.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+import "./Quiz.scss";
+
+class QuizCard extends Component {
+  
+  render() {
+    return (
+      <div className="card">
+        <div className="card-body">
+          <h5 className="card-title">
+            {this.props.title}
+            INTERVIEW PREPARATION
```

Comment:
if the title is coming remove this interview preparation text

Reply:
As there is no title up til now for tutorial or quizzes that's why set this as one